components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OIDCAuthenticatorConstants.java

```diff
@@ -20,8 +20,18 @@
 
 import java.util.regex.Pattern;
 
+/**
+ * This class holds the constants related to the OIDC authenticator.
+ */
 public class OIDCAuthenticatorConstants {
 
+    public static final String STATE_PARAM = "state.param";
+    public static final String ACCESS_TOKEN_PARAM = "accessToken";
+    public static final String CODE_PARAM = "code.param";
+    public static final String ID_TOKEN_PARAM = "idToken";
+    public static final String REQUIRE_NATIVE_CONFIG = "requireNativeConfig";
```

Comment:
Modify the variable value something as requireNativeSDKBasedFederation

Reply:
Since a request parameter is not used to specify whether native SDK based federation is supported, there is no use for `REQUIRE_NATIVE_CONFIG` constant. Thus, it is removed by https://github.com/wso2-extensions/identity-outbound-auth-oidc/commit/3b085d999dc62d7f1f86c36068171e7c2efea6d4